assets/js/googlesitekit/components-gm2/Dialog.js

```diff
@@ -136,9 +135,21 @@ const Dialog = ( {
 							) }
 							<footer className="mdc-dialog__actions">
 								<Button
+									className={ classnames(
```

Comment:
I think it's okay to move the SpinnerButton to the `components-gm2` folder (and to create a placeholder in `components-gm3`) since it is just a variant of the `Button` component.